tracer.go

```diff
@@ -154,10 +154,10 @@ func CreateTracer(opts Options) (Tracer, error) {
 		opentracing.HTTPHeaders: theLightStepPropagator,
 		opentracing.Binary:      theBinaryPropagator,
 	}
-
-	if opts.Propagator == "b3" {
-		impl.propagators[opentracing.TextMap] = theB3Propagator
-		impl.propagators[opentracing.HTTPHeaders] = theB3Propagator
+	if len(opts.Propagators) > 0 {
+		for builtin, propagator := range opts.Propagators {
```

Comment:
For what its worth you don't need the if statement, you can range over a nil map or empty map just fine.

Reply:
good point, fixed